plugins/Wallet/js/components/receiveprompt.js

```diff
@@ -1,15 +1,18 @@
 import React, { PropTypes } from 'react'
 
-const ReceivePrompt = ({address, actions}) => (
-	<div className="modal">
-		<div className="receive-prompt">
-			You can receive Siacoins using the following address:
-			<div className="wallet-address">{ address }</div>
-			<button className="receiveprompt-dismissbtn" onClick={actions.hideReceivePrompt}>OK</button>
+const ReceivePrompt = ({address, actions}) => {
+	const handleDismissClick = () => actions.hideReceivePrompt()
```

Comment:
Yes.  Right now we're not doing anything except dispatching the action, but in the future we could do more (validation for example).  This is also enforced by `"react/jsx-handler-names": 1` in eslintrc.  The alternative would be to rename all click handler actions with the `handle` prefix, or decide on a different prefix to be standard.